src/pages/RealtimeMapPage.tsx

```diff
@@ -131,8 +131,8 @@ export default function RealtimeMapPage() {
             time={to}
             onChange={(ts) => {
               const val = ts ? ts : from
-              setFrom(moment(val))
-              setTo(moment(val).add(moment(to).diff(moment(from)))) // keep the same time difference
+              setFrom(moment(val).subtract(moment(to).diff(moment(from))))
+              setTo(moment(val)) // keep the same time difference
```

Comment:
why do we keep the same time difference?
```suggestion
              setTo(moment(val))
```

Reply:
> why do we keep the same time difference?

If the user sets the difference to 6 minutes and wants to see that difference with another date or change the hour. 
If we lose the time difference we lose the user:)